mon-pix/app/services/url.js

```diff
@@ -60,12 +60,13 @@ export default class Url extends Service {
   }
 
   get accessibilityUrl() {
+    const tld = this.currentDomain.getExtension();
     const currentLanguage = this.intl.t('current-lang');
 
-    if (currentLanguage === 'en') {
-      return `https://pix.${this.currentDomain.getExtension()}/en-gb/accessibility`;
+    if (tld === 'fr') {
+      return `https://pix.fr/accessibilite`;
     }
-    return `https://pix.${this.currentDomain.getExtension()}/accessibilite`;
+    return currentLanguage === 'fr' ? `https://pix.org/accessibilite` : `https://pix.org/accessibility`;
```

Comment:
En testant sur la RA, je vois une erreur 404 sur https://pix.org/accessibility et  https://pix.org/accessibility 🤔 
Ca devrait être plutôt quelque comme ça nn ?
```suggestion
    return currentLanguage === 'fr' ? `https://pix.org/fr/accessibilite` : `https://pix.org/en-gb/accessibility`;
```

Reply:
Oui totalement 🤦🏻‍♀️